config/application.rb

```diff
@@ -15,7 +15,13 @@
 module Pupilfirst
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.1
+    config.load_defaults 7.0
+
+    config.active_support.cache_format_version = 7.0
+
+    config.action_dispatch.cookies_serializer = :hybrid
+
+    config.active_support.disable_to_s_conversion = true
```

Comment:
Similar to `cache_format_version`, this is also set by `load_defaults 7.0`. The instruction in the comments in `new_framework_defaults_7_0.rb` about putting this in `application.rb` applies only when `load_defaults 6.1` is being called.